src/common/NetworkManager.ts

```diff
@@ -238,13 +267,18 @@ export class NetworkManager extends EventEmitter {
       !event.request.url.startsWith('data:')
     ) {
       const requestId = event.requestId;
-      const interceptionId = this._requestIdToInterceptionId.get(requestId);
-      if (interceptionId) {
+      const requestPausedEvent = this._requestIdToRequestPausedEvent.get(
+        requestId
+      );
+
+      this._requestIdToRequestWillBeSentEvent.set(requestId, event);
```

Comment:
In theory, the condition for this line could be stricter:
`!requestPausedEvent || (!this._cacheDisabled() && event.type === 'Font')`
But we don't know for sure if that would cover all cases.